### ISSUE TITLE
training on a new dataset

### ISSUE THREAD

**Issue by MariaLombardi** [Vagver/3DGazeNet#8]
Hello,
I would like to train your model with my own dataset. Any instruction on how to preprocess the dataset, configure and run the algorithm for the training/testing?

Thanks,
Maria

**Comment by eververas** [comment]
Hello,

Does your dataset include ground truth gaze labels? If yes, you need to follow the preprocessing scripts in https://github.com/Vagver/3DGazeNet/tree/main/tools. Note that you can use our pretrained model to get iris landmarks.
